### PR TITLE
The Activity Lifecycle

### PR DESCRIPTION
**Pull request by samvmisti**
The Activity Lifecycle - described main callbacks in short, Added the link to official https://developer.android.com/ site.

### CODE REVIEW

**Review by sud007 — APPROVED** [review]
looks better

File: README.md
Comment (archow) [inline README.md:58]: This is great! Very easy to understand, and I appreciate that you added the source link for more information and cross checking. 
The youtube link does not seem to work anymore however.

Is there a way we can add more admins to this project to help approve changes? Preferably after two or more approvals from Android devs or source-checking.

**Review by archow — APPROVED** [review]

**Review by archow — CHANGES_REQUESTED** [review]
Fix merge conflicts

**Pull request closed without merging**
